genesis/public/core/instance.go

```diff
@@ -93,7 +93,7 @@ func (instDom *instanceDomain) GetInstance(record string) (object.Proxy, error)
 }
 
 type instanceDomainProxy struct {
```

Comment:
instanceDomainProxy is not an implementation of InstanceDomain interface now, but it should be